pkg/controller/datadogagent/service.go

```diff
@@ -149,6 +177,11 @@ func (r *ReconcileDatadogAgent) cleanupMetricsServerService(dda *datadoghqv1alph
 	return cleanupService(r.client, serviceName, dda.Namespace)
 }
 
+func (r *ReconcileDatadogAgent) cleanupMetricsServerAPIService(dda *datadoghqv1alpha1.DatadogAgent) (reconcile.Result, error) {
+	apiServiceName := getMetricsServerServiceName(dda)
```

Comment:
same as the createAPIService, could we add a log:
`logger.Info("Delete APIService", "name", apiServiceName)` ? (maybe below in case it's not found)

Reply:
Yes, will add these.